common/buildcraft/lib/fake/FakePlayerBC.java

```diff
@@ -20,5 +20,6 @@ public FakePlayerBC(WorldServer world, GameProfile name) {
 
     @Override
     public void openEditSign(TileEntitySign signTile) {
+        // TODO: Put this in forge!
```

Comment:
want me to make a forge PR for this? if so in why are we overriding this? when does/could this get called for fake players? automated clicker (stripe pipe) pointed at a sign i presume?

Reply:
The builder (or filler?) placing signs automatically. The default player method puts the player into the sign tile entity, so removing it was necessary.

Reply:
alright, i'll see to make a quick forge PR for this

Reply:
The last time I looked at this it could be avoided by placing the stack differently... seeing as how signs need special support we might be able to avoid this completely. A forge PR might not be a bad idea though.